CHANGELOG.md

```
@@ -2,44 +1,31 @@
-
-
-### Features
+## Features
 
 * **#144:** remove hardcoding in .env ([#145](https://github.com/VilnaCRM-Org/user-service/issues/145)) ([cf0ddb8](https://github.com/VilnaCRM-Org/user-service/commit/cf0ddb83c9794387d1c2dedcf6e56529931dd5a9)), closes [#144](https://github.com/VilnaCRM-Org/user-service/issues/144) [#144](https://github.com/VilnaCRM-Org/user-service/issues/144) [#144](https://github.com/VilnaCRM-Org/user-service/issues/144)
 
-
-
 ## [2.3.1](https://github.com/VilnaCRM-Org/user-service/compare/v2.3.0...v2.3.1) (2025-02-04)
 
+## Bug Fixes
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)